python/tank/authentication/session.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) 2017 Autodesk.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+"""
+RV Session abstraction.
+
+The main purpose of this module is to avoid having to constantly fecth either
+the RV session or read cookies from the QWebView used for the login process.
+
+Other than from hardcoding the RV Session format in one method, we have
+attempted to keep this module relatively independant.
+"""
+
+
+class Session(object):
```

Comment:
this class name feels too generic - can it made more specific? `AuthenicationSessionData` or something?